File: docs/examples/record_api_rs/src/create.rs
use trailbase_client::Client;

pub async fn create(client: &Client) -> anyhow::Result<String> {
  Ok(
    client
      .records("simple_strict_table")
      .create(serde_json::json!({
          "text_not_null": "test",
      }))
      .await?,
  )
}

File: docs/examples/record_api_go/create.go
package record_api_docs

import "github.com/trailbaseio/trailbase/client/go/trailbase"

func Create(client trailbase.Client) (trailbase.RecordId, error) {
	api := trailbase.NewRecordApi[SimpleStrict](client, "simple_strict_table")
	return api.Create(SimpleStrict{
		TextNotNull: "test",
	})
}
